client/src/components/layout/Sidebar.js

```diff
@@ -0,0 +1,36 @@
+import { Navbar, Nav, Image } from "react-bootstrap"
```

Comment:
All these imports are unused according to the static analyzer, can remove if not needed